frontend/src/components/NewShoot/NewShootInfrastructureDetails.vue

```diff
@@ -96,6 +96,24 @@ limitations under the License.
           ></v-select>
         </v-flex>
       </template>
+      <template v-else-if="infrastructureKind === 'vsphere'">
```

Comment:
persistent hint: Default Load Balancer Class is required

Reply:
✔️